src/Products/Annotation/Annotator/PolylineAnnotator.cs

```diff
@@ -1,81 +1,81 @@
 using System;
 
 namespace GroupDocs.Annotation.MVC.Products.Annotation.Annotator
 {
     public class PolylineAnnotator : BaseAnnotator
     {
-        public PolylineAnnotator(AnnotationDataEntity annotationData, PageData pageData)
-            : base(annotationData, pageData)
-        {
-        }
-        
-        public override AnnotationInfo AnnotateWord()
+        private PolylineAnnotation polylineAnnotation;
+
+        public PolylineAnnotator(AnnotationDataEntity annotationData, PageInfo pageInfo)
+            : base(annotationData, pageInfo)
         {
-            AnnotationInfo polylineAnnotation = InitAnnotationInfo();
-            return polylineAnnotation;
+            this.polylineAnnotation = new PolylineAnnotation
+            {
+                Box = GetBox(),
+                PenColor = 1201033,
+                PenWidth = 2,
+                SvgPath = annotationData.svgPath
+            };
         }
-        
-        public override AnnotationInfo AnnotatePdf()
+
+        public override AnnotationBase AnnotateWord()
         {
-            AnnotationInfo polylineAnnotation = InitAnnotationInfo();
+            polylineAnnotation = InitAnnotationBase(polylineAnnotation) as PolylineAnnotation;
             return polylineAnnotation;
         }
-        
-        protected new AnnotationInfo InitAnnotationInfo()
+
+        public override AnnotationBase AnnotatePdf()
         {
-            AnnotationInfo polylineAnnotation = base.InitAnnotationInfo();
-            polylineAnnotation.PenColor = 1201033;
-            polylineAnnotation.PenWidth = (byte)2;
-            polylineAnnotation.SvgPath = annotationData.svgPath;
+            polylineAnnotation = InitAnnotationBase(polylineAnnotation) as PolylineAnnotation;
             return polylineAnnotation;
         }
-        
-        public override AnnotationInfo AnnotateCells()
+
+        public override AnnotationBase AnnotateCells()
         {
-            throw new NotSupportedException(String.Format(Message, annotationData.type));
+            throw new NotSupportedException(string.Format(Message, annotationData.type));
         }
-        
-        public override AnnotationInfo AnnotateSlides()
+
+        public override AnnotationBase AnnotateSlides()
         {
-            AnnotationInfo polylineAnnotation = InitAnnotationInfo();
-            fillCreatorName(polylineAnnotation);
+            polylineAnnotation = InitAnnotationBase(polylineAnnotation) as PolylineAnnotation;
+            FillCreatorName(polylineAnnotation);
             return polylineAnnotation;
         }
 
         /// <summary>
         /// Fill creator name field in annotation info
         /// </summary>
-        /// <param name="polylineAnnotation">AnnotationInfo</param>
-        protected void fillCreatorName(AnnotationInfo polylineAnnotation)
+        /// <param name="polylineAnnotation">AnnotationBase</param>
+        protected void FillCreatorName(AnnotationBase polylineAnnotation)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'FillCreatorName' a static method.](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372471)